### PR TITLE
added modal

### PR DESCRIPTION
**Pull request by anshikaa12**
modaldoc

### CODE REVIEW

File: components/modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    bg1.style.display="none";
+}
+close_btnc.onclick=()=>{
+    clickanywhere.classList.add("hide-modal");
+    bg2.style.display="none";
+}
+/******for click anywhere********* */
+window.onclick = function(event) {
+    if (event.target == bg2) {
+      clickanywhere.classList.add("hide-modal");
```
Comment (Manpreet-hub) [inline components/modal/modal.js:29]: suggestion: I think you should use here tripe equals operator.

**Review by Manpreet-hub — COMMENTED** [review]
Code looks good. Just one suggestion added.

File: components/modal/modal.js (hunk 1 above)
Comment (anshikaa12) [reply components/modal/modal.js:29]: why so?

Comment (anshikaa12) [reply components/modal/modal.js:29]: i mean it works fine with == also? so whats the need?

Comment (ivinayakg) [reply components/modal/modal.js:29]: the need is === is a safer option as you can be sure that the value, as well as the type of the data, is the same, it wont affect the code here maybe somewhere else but it's a good practice to do so.
you can also read more about this...

**Review by ivinayakg — APPROVED** [review]
code looks good, go ahead and merge it.

**Review by Manpreet-hub — APPROVED** [review]

**Pull request merged**